<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCharaPicker</class>
 <widget class="QDialog" name="DialogCharaPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>289</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>412</width>
    <height>289</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>412</width>
    <height>297</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Character sprite picker</string>
  </property>
  <layout class="QGridLayout" name="layoutCharaPicker">
   <item row="0" column="0">
    <widget class="QListWidget" name="listRes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedMain">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageChara">
      <layout class="QVBoxLayout" name="layoutPageChara">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layoutChara">
         <item>
          <spacer name="spacerCharaLeft">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGraphicsView" name="viewChara">
           <property name="minimumSize">
            <size>
             <width>198</width>
             <height>134</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>198</width>
             <height>134</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerCharaRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedStatic">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageStatic">
          <layout class="QHBoxLayout" name="layoutStatic">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupFacing">
             <property name="title">
              <string>Facing</string>
             </property>
             <layout class="QGridLayout" name="layoutFacing">
              <item row="0" column="1">
               <widget class="QToolButton" name="toolUp">
                <property name="text">
                 <string>U</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QToolButton" name="toolLeft">
                <property name="text">
                 <string>L</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QToolButton" name="toolRight">
                <property name="text">
                 <string>R</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QToolButton" name="toolDown">
                <property name="text">
                 <string>D</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupFrame">
             <property name="title">
              <string>Frame</string>
             </property>
             <layout class="QVBoxLayout" name="layoutFrame">
              <item>
               <widget class="QRadioButton" name="radioLeft">
                <property name="text">
                 <string>Left</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioMiddle">
                <property name="text">
                 <string>Middle</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioRight">
                <property name="text">
                 <string>Right</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageEmpty"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTileset">
      <layout class="QVBoxLayout" name="layoutTileset">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="graphicsTile">
         <property name="maximumSize">
          <size>
           <width>212</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCharaPicker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCharaPicker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
